extensions/amp-sidebar/0.1/amp-sidebar.md

```diff
@@ -326,6 +326,49 @@ When using `toolbar` feature, `autoscroll` only works if the `<nav toolbar>` ele
 
 Please see [this example file](https://github.com/ampproject/amphtml/blob/master/examples/amp-sidebar-autoscroll.amp.html) for a working example code.
 
+## Building Nested Menus
+
+`<amp-sidebar>` can supports drilldown (nested) menus through a child component named [`<amp-nested-menu>`](https://amp.dev/documentation/components/amp-nested-menu/). With `<amp-nested-menu>`, `<amp-sidebar>` can support nesting one or more layers of submenus (and transition between them) as demonstrated by the following example:
```

Comment:
Please update the links to relative GitHub ones - the amp.dev build process converts them on the amp.dev side :)